clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -536,12 +536,28 @@ class CIRMemCpyOpLowering
   }
 };
 
+static mlir::Value getLLVMVecCast(mlir::ConversionPatternRewriter &rewriter,
```

Comment:
Looks like you forgot to remove this function?

Reply:
removed in ready-to-review version